packages/server/prisma/migrations/20221108134009_init/migration.sql

```diff
@@ -0,0 +1,7 @@
+-- CreateTable
+CREATE TABLE "Test" (
```

Comment:
Is this Test-table something we want to merge to main?

Reply:
Yeah this model is unnecessary